projects/manager/src/app/permissions/permissions.component.html

```diff
@@ -14,10 +14,51 @@
         <td mat-cell *matCellDef="let element">{{ element.permission }}</td>
     </ng-container>
 
+    <ng-container matColumnDef="remove">
+        <th mat-header-cell *matHeaderCellDef>Remove</th>
+        <td mat-cell *matCellDef="let element">
+            <button mat-icon-button (click)="removePermission(element)">
+                <mat-icon>delete</mat-icon>
+            </button>
+        </td>
+    </ng-container>
+
     <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
     <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
 </table>
 
 <mat-paginator [pageSize]="5" [pageSizeOptions]="[5, 10, 20]"></mat-paginator>
 
+<form [formGroup]="form">
+    <div class="flex-container">
+        <mat-form-field>
+            <mat-label>Role Specification</mat-label>
+            <mat-select formControlName="roleSpec">
+                <mat-option class="options" *ngFor="let roleSpec of ROLE_SPECS" [value]="roleSpec">
+                    {{ roleSpec }}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <mat-form-field appearance="fill">
+            @if (form.get('roleSpec')?.value === ROLE_SPEC.Id) {
```

Comment:
What is this?

Can we wrap this, e.g., into a function so that we will know in a month what we are doing here? :D

Reply:
I removed adding by id 5f3f7d5d3e2d563fae64e15c23f4a3536d78a2db